docs/en/compute/Hive.md

```diff
@@ -268,7 +268,7 @@ hive> alter table u_user set location "alluxio:///ml-100k";
 Since Alluxio 2.0, one can directly use Alluxio HA-style authorities in Hive queries without any configuration setup.
 See [HA authority]({{ '/en/deploy/Running-Alluxio-On-a-HA-Cluster.html' | relativize_url }}#ha-authority) for more details.
 
-### Experimental: Use Alluxio as the Default Filesystem
```

Comment:
This line was from 2018. I guess not "experimental" any more?

Reply:
Agree that we can remove this word

Reply:
I talked to Gene, seems this is still "experimental". So i brought it back.